src/tmpfiles/tmpfiles.c

```diff
@@ -2718,6 +2886,13 @@ static int parse_line(
         if (r < 0)
                 return r;
 
+        /*
+         * The "age-by" argument considers all
+         * file timestamp types by default.
```

Comment:
we usually line break at 109 chars btw... see docs/CODING_STYLE.md

Reply:
ACK.